app/models/micropost.rb

```diff
@@ -0,0 +1,21 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+
+  validates :user_id, presence: true
```

Comment:
Tìm hiểu lại đi. đọc cái này xem có rút ra được gì không.
http://railsguides.net/belongs-to-and-presence-validation-rule1/